Replace `myst-parser` with `pandoc`.

The former isn’t available on bullseye/byzantium but the latter is.
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -60,6 +60,3 @@
 
       - name: Run unit tests
         run: poetry run poe tests
-
-      - name: Run man page and HTML documentation generator
-        run: poetry run poe doc
--- a/.gitignore
+++ b/.gitignore
@@ -5,6 +5,7 @@
 /debian/itchcraft/
 /debian/itchcraft.*.debhelper
 /debian/itchcraft.substvars
+/debian/*.rst
 /dist/
 /doc/sphinx/api/
 /.pc/
--- a/doc/sphinx/conf.py
+++ b/doc/sphinx/conf.py
@@ -22,7 +22,6 @@
 
 extensions = [
     'autoapi.extension',
-    'myst_parser',
     'sphinx.ext.autodoc',
 ]
 
--- a/doc/sphinx/index.rst
+++ b/doc/sphinx/index.rst
@@ -8,5 +8,4 @@
    usage
    api/index
 
-.. include:: ../../README.md
-   :parser: myst_parser.sphinx_
+.. include:: ../../debian/README.rst
--- a/doc/sphinx/usage.rst
+++ b/doc/sphinx/usage.rst
@@ -1,5 +1,4 @@
 Usage
 =====
 
-.. include:: ../../USAGE.md
-   :parser: myst_parser.sphinx_
+.. include:: ../../debian/USAGE.rst
